build.sbt

```diff
@@ -358,3 +352,16 @@ ThisBuild / githubWorkflowBuildPostamble ++= Seq(
     name = Some("Upload Codecov Results")
   )
 )
+
+def crossCommand(command: String) =
+  List(s"++$Scala212", s"root/$command", s"++$Scala213", s"root-spark32/$command")
```

Comment:
Hm, what would be the release artifacts names? 

Before these changes we had the following artefacts naming strategy (copied it from the README.md):

Artifact names examples:

* frameless-dataset (the latest Spark dependency)
* frameless-dataset-spark31 (Spark 3.1.x dependency)
* frameless-dataset-spark30 (Spark 3.0.x dependency)

Critical comments: 
* We didn't publish `-spark32` before the #597 
* I'm okay with publishing `-spark32` (though it increases the build time), but let's be sure that the _non prefixed artifact_ is published for the _latest Spark (3.2)) and cross 2.12 / 2.13 Scala versions. This is extremely important since the majority of users probably depend on the non prefixed artifact and that's the way then can get updates.

Reply:
Good question. The artifact prefixes haven't changed because they are determined by the `name` setting.
https://github.com/typelevel/frameless/blob/64698754762f8475e676c7c5bcf27044bd1f7189/build.sbt#L64
https://github.com/typelevel/frameless/blob/64698754762f8475e676c7c5bcf27044bd1f7189/build.sbt#L70
https://github.com/typelevel/frameless/blob/64698754762f8475e676c7c5bcf27044bd1f7189/build.sbt#L78

Reply:
Oh right I forgot, Thanks!